pkg/migrations/mls/20250707105710_add_is_commit.up.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE group_messages ADD COLUMN is_commit BOOLEAN DEFAULT NULL;
+
+CREATE OR REPLACE FUNCTION insert_group_message(
```

Comment:
One thing I’ve learned the hard way is that we should always create new functions instead of replacing. 

That way when a deploy is rolled out halfway the old code can work. Also makes reverts a simple code change, with no DB rollback required.